schemas/primary_diagnosis.json

```diff
@@ -230,7 +230,11 @@
           "Weight Loss"
         ]
       },
-      "meta": { "displayName": "Presenting Symptoms" }
+      "meta": { 
+         "displayName": "Presenting Symptoms",
+         "notes": "To include multiple values, please separate values with a comma ',' within your file",
+        "examples": "Anemia,Bloating,Diabetes"
```

Comment:
I may just say 
For example: Anemia, Bloting, Diabetes

As it's just one example

Reply:
oh wait, never mind - is the word "examples" a dictionary variable thing